db/fixtures/miq_product_features.yml

```diff
@@ -339,6 +339,14 @@
       :description: Display Individual Cloud Providers
       :feature_type: view
       :identifier: ems_cloud_show
+    - :name: Arbitration Profiles List
```

Comment:
i think renaming feature ids is ok right now as these feature ids are not being used until this PR gets merged.